assets/js/squad.js

```diff
@@ -56,13 +56,13 @@ jQuery(function ($) {
       } catch (error) {
         res = response;
       }
-      const responseTransactionRef = res.transaction_ref;
+      const respTxnref = res.transaction_ref;
```

Comment:
naming convention is a bit vague, let's use the former or change to something more readable 😀